charts/shiny-app/templates/fluentd-configmap.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ .Chart.Name }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: "{{ template "fullname" . }}-fluentd"
+data:
+  elasticsearch_host: "{{ .Values.Elasticsearch.Host }}"
```

Comment:
Where these `Values.Elasticsearch.*` values come from?

I guess they can be passed via command line arguments and have good defaults in the env shiny-app values but I can't see them.

It may be worth to add blank values in the default values YAML and/or a note in the shiny-app README.